sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Generated/Models/KafkaRestProperties.cs

```diff
@@ -35,7 +35,7 @@ public KafkaRestProperties()
         /// <param name="clientGroupInfo">The information of AAD security
         /// group.</param>
         /// <param name="configurationOverride">The configurations that need to
-        /// be overriden.</param>
```

Comment:
If the file is under Generated folder, we can't simply change it like that. A swagger change is required and re-run the generate.ps1 to generate the sdk.